app/src/main/java/com/sdp/movemeet/MainActivity.java

```diff
@@ -42,16 +42,11 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        //phone = findViewById(R.id.text_view_profile_phone);
```

Comment:
Was it just to test something out, or do you really don't need those user information anymore in the MainActivity?

Reply:
I had just forgotten it, I deleted it ;)

Reply:
In fact, if I understood correctly, the information is retrieved through the method handleRegisterUser();